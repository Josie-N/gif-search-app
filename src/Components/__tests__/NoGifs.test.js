import * as React from "react";
import NoGifs from "../NoGifs";
import { render } from "@testing-library/react";

describe("NoGifs component", () => {
  describe("when it loads", () => {
    it("renders component", () => {
      const { container } = render(<NoGifs />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
