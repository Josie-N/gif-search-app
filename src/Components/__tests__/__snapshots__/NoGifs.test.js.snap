// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoGifs component when it loads renders component 1`] = `
<li
  class="no-gifs"
>
  <i
    class="material-icons icon-gif"
  >
    sentiment_very_dissatisfied
  </i>
  <h3>
    Sorry, no GIFs match your search.
  </h3>
</li>
`;
